packages/react/src/components/header/Header.stories.tsx

```diff
@@ -303,3 +466,37 @@ export const WithSkipLink = (args) => {
     </Header>
   );
 };
+
+export const WithSkipLinkInDarkMode = (args) => {
```

Comment:
Maybe just one story with dark theme is enough? Maybe the full features one so every component is in display. And slap the SkipLink there as well. I think it's good to not have the stories section too cluttered and not with too similar stories.